@xen-orchestra/web/src/components/vms/panel/cards/VmInfoCard.vue

```diff
@@ -0,0 +1,199 @@
+            <VtsObjectIcon type="host" :state="hostPowerState" size="small" />
+            <div v-tooltip class="text-ellipsis">
+              <UiLink :to="`/host/${host.id}`" size="small">
+                {{ host.name_label }}
+              </UiLink>
+            </div>
+            <VtsIcon v-if="isMaster" v-tooltip="t('master')" name="legacy:primary" size="small" />
+          </div>
+        </template>
+        <template v-if="host" #addons>
+          <VtsCopyButton :value="host.name_label" />
+        </template>
+      </VtsCardRowKeyValue>
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('os-name') }}</template>
+        <template #value>{{ vm.os_version?.name }}</template>
+        <template v-if="vm.os_version?.name" #addons>
+          <VtsCopyButton :value="vm.os_version?.name" />
+        </template>
+      </VtsCardRowKeyValue>
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('guest-tools') }}</template>
+        <template #value>
+          <div class="guest-tools">
+            <VtsIcon
+              v-if="guestToolsDisplay.value !== '-'"
+              v-tooltip="guestToolsDisplay.tooltip"
+              :name="guestToolsDisplay.type === 'link' ? 'legacy:halted' : 'legacy:checked'"
+              size="medium"
+            />
+            <template v-if="guestToolsDisplay.type === 'link'">
+              <UiLink size="small" href="https://docs.xcp-ng.org/vms/#guest-tools">
+                {{ guestToolsDisplay.value }}
+              </UiLink>
+            </template>
+            <template v-else>
+              {{ guestToolsDisplay.value }}
+            </template>
+          </div>
+        </template>
+      </VtsCardRowKeyValue>
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('template') }}</template>
+        <template #value>
+          <div class="template">
+            <VtsIcon name="fa:template" size="medium" />
+            {{ vm.other.base_template_name }}
+          </div>
+        </template>
+        <template v-if="vm.other.base_template_name" #addons>
+          <VtsCopyButton :value="vm.other.base_template_name" />
+        </template>
+      </VtsCardRowKeyValue>
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('created-on') }}</template>
+        <template #value>{{ installDateFormatted }}</template>
+        <template v-if="installDateFormatted" #addons>
+          <VtsCopyButton :value="installDateFormatted" />
+        </template>
+      </VtsCardRowKeyValue>
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('created-by') }}</template>
+        <template #value>
+          <div class="user">
+            <UiUserLogo size="extra-small" />
+            {{ user?.email ?? t('unknown') }}
+          </div>
+        </template>
+        <template v-if="user?.email" #addons>
+          <VtsCopyButton :value="user?.email" />
+        </template>
+      </VtsCardRowKeyValue>
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('started') }}</template>
```

Comment:
```suggestion
        <template #key>{{ t('vm-uptime') }}</template>
```

In Figma, **started** is replaced by **VM uptime** 

<img width="441" height="114" alt="image" src="https://github.com/user-attachments/assets/166b6285-a0b2-44c8-936b-da68e28ac1ca" />

@pdonias @clemencebx In vm dashboard the label is **started** , so what is the right label ?